x-pack/plugins/security_solution/public/landing_pages/constants.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { SecurityPageName } from '../app/types';
+
+export interface LandingNavGroup {
+  label: string;
+  itemIds: SecurityPageName[];
+}
+
+export const MANAGE_NAVIGATION_CATEGORIES: LandingNavGroup[] = [
```

Comment:
Is this used somewhere? Does it means rules and expections will be under Management section?

Are the `itemIds` array sorted? If are, `itemIds` for Endpoints should be like this I guess:

```Typescript
[
      SecurityPageName.endpoints,
      SecurityPageName.policies,
      SecurityPageName.trustedApps,
      SecurityPageName.eventFilters,
      SecurityPageName.hostIsolationExceptions,
      SecurityPageName.blocklist,
    ]
```

Reply:
> Is this used somewhere? Does it means rules and expections will be under Management section?

It is used for the menu landing page. Yes, rules and exceptions are located on the Manage landing page.

<img width="810" alt="Screenshot 2022-05-17 at 10 32 55" src="https://user-images.githubusercontent.com/1490444/168768083-cd98ff07-8af7-4159-98bb-d651530d1542.png">



<hr />
 

> Are the itemIds array sorted? If they are, itemIds for Endpoints should be like this I guess:

hmm, I see that on your list, `hostIsolationExceptions` comes before `blocklist`. But in the design, it is the other way around.

<img width="895" alt="Screenshot 2022-05-17 at 10 09 27" src="https://user-images.githubusercontent.com/1490444/168768921-9590200e-6e05-41d7-9174-5d77a75e8b8a.png">


@bfishel Is it ok if we follow the order proposed by @dasansol92 instead the one on Figma?

Reply:
Actually, it is `Blocklist` and `Event filters` that have messages swapped 😅